src/towncrier/test/test_check.py

```diff
@@ -190,51 +196,68 @@ def test_none_stdout_encoding_works(self):
         self.assertEqual(b"", stderr)
 
     def test_first_release(self):
-        """The checks should be skipped on a branch that creates the news file.
+        """
+        The checks should be skipped on a branch that creates the news file.
 
-        If the checkes are not skipped in this case, towncrier check would fail
-        for the first release that has a changelog"""
+        If the checks are not skipped in this case, towncrier check would fail
+        for the first release that has a changelog.
+        """
         runner = CliRunner()
 
         with runner.isolated_filesystem():
-            # Arrange
-            write("towncrier.toml", "[tool.towncrier]")
-            write("newsfragments/123.feature", "Foo the bar")
-            initial_commit()
+            create_project()
+            # Before any release, the NEWS file might no exist.
+            self.assertNotIn('NEWS.rst', os.listdir())
 
-            call(["git", "checkout", "-b", "next-resease"])
```

Comment:
The intention is to try to reuse the `create_project` so that the tests will use the same same set of conventions when managing the news files. 

So, to increase consistency.

I am not happy with `create_project`, but the idea is that when you work on these tests, you understand what `create_project` does once, and then you should have a good idea of what to expect in each test.